in-cluster/default.yaml

```diff
@@ -496,3 +496,15 @@ spec:
           - pass:
               when: "false"
               message: No OOMKilling event detected
+    - textAnalyze:
+        checkName: "CoreDNS nameserver configuration"
+        regex: "no nameservers found"
+        ignoreIfNoFiles: true
+        fileName: podlogs/coredns/coredns-*/coredns.log
```

Comment:
@nvanthao small fix: in this spec, I don't think we're using the `podlogs/` path:

```
- textAnalyze:
        checkName: Minio disk full
        fileName: cluster-resources/pods/logs/kurl/registry-*/registry.log
```

that should probably be `/cluster-resources/pods/logs/kube-system/coredns-*/coredns.log`

Reply:
many thanks @adamancini, I've made the update!